src/ralph/attachments/views.py

```diff
@@ -0,0 +1,50 @@
+from django.forms.models import modelformset_factory
+from django.http import HttpResponseRedirect
+
+from ralph.admin.views.extra import RalphDetailView
+from ralph.attachments.forms import AttachmentForm
+from ralph.attachments.models import Attachment, AttachmentItem
+
+
+class AttachmentsView(RalphDetailView):
+    icon = 'paperclip'
+    name = 'attachment'
+    label = 'Attachments'
+    url_name = 'attachment'
+    template_name = 'attachments/attachments.html'
+
+    def get_formset(self, data=None, files=None):
+        qs = Attachment.objects.get_attachments_for_object(self.object)
+        AttachmentForm._request = self.request
```

Comment:
isn't there better way to handle this? assigning to class it's not the best idea. Maybe you could use `RalphAdminForm` (https://github.com/allegro/ralph/blob/ng/src/ralph/admin/mixins.py#L82)?